packages/launchpad/src/checklist-item/style.scss

```diff
@@ -25,6 +24,14 @@
 	text-decoration: none;
 	padding-left: 2px;
 
+	&:is(button, a) {
+		cursor: pointer;
+	}
+
+	&:is(div) {
+		cursor: default;
```

Comment:
It needs to be overridden because it comes with the `cursor: pointer` from `@wordpress/components`.